mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -97,9 +101,13 @@ def binary_cross_entropy(pred,
     """
     # The default value of ignore_index is the same as F.cross_entropy
     ignore_index = -100 if ignore_index is None else ignore_index
-    if pred.dim() != label.dim():
-        label, weight = _expand_onehot_labels(label, weight, pred.size(-1),
-                                              ignore_index)
+    if expand_one_hot:
```

Comment:
In Mask2former, labels passed to bce is  a float (between 0 and 1), not a interger. The interface is for this purepose. Besides, by set `expand_one_hot` to `False`, we can directly pass `gt_labels` into function instead of `(1 - gt_labels)`, but this setting was not used in maskformer_head yet.

Reply:
It should not be added in this pr. Isolation between different PRs is necessary.

Reply:
Do not forget to deal with this comment.